scripts/index.js

```diff
@@ -13,7 +13,9 @@ function getLocationLongitude(obj) {
     return obj.coord.lon;
 }
 function getDescription(obj) {
-    return obj.weather[0].description;
+    return obj.weather.map(function (each) {
```

Comment:
Nice. This will handle 0 or many "chunks" of weather data. On the other hand, this won't give you a single string, and won't pass the test. 

But, in this case, I would say that it's a good opportunity to update the tests. :)